docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -114,11 +116,11 @@ To enable your application to use the media content functionality:
     ```
 
 
-## Finding Albums
+## Find albums
 
 To find the albums satisfying certain criteria, or modify the search results in a specific way, create a filter and set its properties.
 
-The following example filters media albums so that only albums with the artist named "Tizen" are included in the result. The filter is case-insensitive, and the results are sorted in descending order by album name. For more information on the filter properties, see [Setting up a Filter](#setting-up-a-filter).
+The following example filters media albums so that only albums with the artist named "Tizen" are included in the result. The filter is case-insensitive, and the results are sorted in descending order by album name. For more information on the filter properties, see [Set up a Filter](#set-up-a-filter).
```

Comment:
```suggestion
The following example filters media albums so that only albums with the artist named "Tizen" are included in the result. The filter is case-insensitive, and the results are sorted in descending order by album name. For more information on the filter properties, see [set up a filter](#set-up-a-filter).
```